src/server/multi_command_squasher.cc

```diff
@@ -15,6 +16,8 @@
 #include "server/transaction.h"
 #include "server/tx_base.h"
 
+ABSL_FLAG(size_t, throttle_squashed, 0, "");
```

Comment:
@adiholden I will adjust as we said f2f. Looking for some early feedback based on our discussion